lms/services/lti_outcomes.py

```diff
@@ -65,13 +63,10 @@ def read_result(self, outcomes_request_params):  # pylint:disable=no-self-use
         )
 
         try:
-            score = self.find_element(
-                result, ["readResultResponse", "result", "resultScore", "textString"]
+            return float(
+                result["readResultResponse"]["result"]["resultScore"]["textString"]
             )
-            if score is None:
-                return None
-            return float(score.text)
-        except (ValueError, TypeError):
+        except (KeyError, ValueError):
```

Comment:
Ok, so even though we're now using dicts to _render_ XML, we were still using XPath to _parse_ XML and pull certain parts out of it. Now, we use xmltodict to parse the XML into a dict and then just pull things out of the dict.